### PR TITLE
Add new exceptions and syntax from Python 3.5.

### PR DESCRIPTION
**Pull request by zvezdan**
Thanks to Yuri Selivanov for the original proposal of this patch.

### CODE REVIEW

File: runtime/syntax/python.vim [hunk 1]
```diff
@@ -80,6 +81,7 @@ syn keyword pythonRepeat	for while
 syn keyword pythonOperator	and in is not or
 syn keyword pythonException	except finally raise try
 syn keyword pythonInclude	from import
+syn keyword pythonAsync		async await
```
Comment (ZyX-I) [inline runtime/syntax/python.vim:84]: You have tabs here used for alignment, but they put keywords too far. Note that in Vim you need to set `ts=8` to get correct view. GH also uses ts=8 here.

**Comment by chrisbra** [comment]
PR are not used in this repository. Please send an updated syntax file to Bram for distribution with Vim. Thanks.

File: runtime/syntax/python.vim (hunk 1 above)
Comment (zvezdan) [inline runtime/syntax/python.vim:84]: If you take a look at the whole file you'll notice this at the end:
" vim:set sw=2 sts=2 ts=8 noet:
Therefore, the tabs _are_ set to 8 (as they should) and the alignment works exactly as I intended.
This is a non-issue.
Thanks!

**Comment by zvezdan** [comment]
Thanks! I'll close the PR.
I sent the diff to Bram and Cc to vim-dev.

**Pull request closed without merging**